Makefile

```diff
@@ -0,0 +1,81 @@
+.PHONY: $(filter-out vendor bin/tools/psalm/vendor bin/tools/cs-fixer/vendor,$(MAKECMDGOALS))
+
+MYSQL_URL="mysql://root:root@mysql:3306/zenstruck_foundry?charset=utf8"
+POSTGRESQL_URL="postgresql://root:root@postgres:5432/zenstruck_foundry?charset=utf8"
+MONGO_URL="mongodb://mongo:mongo@mongo:27017/mongo?compressors=disabled&amp;gssapiServiceName=mongodb&authSource=mongo"
+SQLITE_URL="sqlite://./var/app.db"
+
+INTERACTIVE:=$(shell [ -t 0 ] && echo 1)
+ifdef INTERACTIVE
+	DC_EXEC=docker-compose exec
+else
+	DC_EXEC=docker-compose exec -T
+endif
+
+DOCKER_PHP=${DC_EXEC} php
+
+.DEFAULT_GOAL := help
+
+UID = $(shell id -u)
+
+DOCKER_IS_UP = $(shell docker-compose ps | grep "Up (healthy)")
+
+ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+$(eval $(RUN_ARGS):;@:)
+
+help:
+	@fgrep -h "###" $(MAKEFILE_LIST) | fgrep -v fgrep | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+tests: cs-fixer psalm phpunit-full ### Run full tests suite
+
+phpunit-full: vendor ### Run full PHPunit (MySQL + Mongo)
```

Comment:
I'm still questioning the need for all the permutations. I was sort of thinking just `test-full` (mongo/mysql in docker) and `test-fast` (sqlite no docker).

I don't feel super strongly about this but do you think it's important to have all these?

Reply:
what if the CI fails on postgre? the user won't be able to reproduce...

Reply:
yes, why not... maybe I should update the "how to contribute" to explain this